### PR TITLE
Fixed Bug: Consistent model structure

### LINKED ISSUE #35: Can't import basic.utils.get_active_filter()
Sir the models structure has incocnsistency as filter considers param initDate and finDate while news model has param name StartDate and endDate. And it wasn't able to import basic.utils.get_active_filter().

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes #35

### CODE REVIEW

**Comment by yashLadha** [comment]
Sir, the static method is not serializable that's why i have used module scope function for setting default values. And `basic.util.get_basic_filter()` is throwing error so i had changed import statements. Please review it again Sir, @singhpratyush

File: wsgi/odyssy/announcement/models.py [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-from django.db.models import Q
+
+
+def get_current_date():
+    """ set default as start of date """
+    return datetime.combine(datetime.today(), time.min)
+
+def get_final_date():
+    """ set end as undefined date """
+    return datetime.combine(datetime.max, time.max)
```
Comment (singhpratyush) [inline wsgi/odyssy/announcement/models.py]: Make these methods as `get_today_start` and `get_today_end` and move them to `utils` too. Make changes wherever needed.

**Comment by yashLadha** [comment]
Sir every time when we make changes to the model, and we perform makemigrations. We have to first explicitly remove previous migration files to work correctly. Is there a fix to this. Else all things are working perfectly fine.

File: wsgi/odyssy/announcement/models.py (hunk 1 above)
Comment (yashLadha) [inline wsgi/odyssy/announcement/models.py]: :+1: Done

**Pull request merged**